src/test/java/org/opentripplanner/routing/algorithm/mapping/RaptorPathToItineraryMapperTest.java

```diff
@@ -0,0 +1,147 @@
+package org.opentripplanner.routing.algorithm.mapping;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import java.time.Instant;
+import java.time.ZoneId;
+import java.util.ArrayList;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.opentripplanner.model.PickDrop;
+import org.opentripplanner.model.StopTime;
+import org.opentripplanner.routing.algorithm.raptoradapter.transit.cost.DefaultCostCalculator;
+import org.opentripplanner.routing.algorithm.raptoradapter.transit.cost.RaptorCostConverter;
+import org.opentripplanner.routing.api.request.RouteRequest;
+import org.opentripplanner.routing.graph.Graph;
+import org.opentripplanner.transit.model.basic.TransitMode;
+import org.opentripplanner.transit.model.framework.FeedScopedId;
+import org.opentripplanner.transit.model.network.Route;
+import org.opentripplanner.transit.model.network.StopPattern;
+import org.opentripplanner.transit.model.network.TripPattern;
+import org.opentripplanner.transit.model.organization.Agency;
+import org.opentripplanner.transit.model.site.RegularStop;
+import org.opentripplanner.transit.raptor._data.api.TestPathBuilder;
+import org.opentripplanner.transit.raptor._data.transit.TestAccessEgress;
+import org.opentripplanner.transit.raptor._data.transit.TestRoute;
+import org.opentripplanner.transit.raptor._data.transit.TestTransitData;
+import org.opentripplanner.transit.raptor._data.transit.TestTripPattern;
+import org.opentripplanner.transit.raptor._data.transit.TestTripSchedule;
+import org.opentripplanner.transit.raptor.api.path.Path;
+import org.opentripplanner.transit.raptor.api.transit.CostCalculator;
+import org.opentripplanner.transit.service.DefaultTransitService;
+import org.opentripplanner.transit.service.TransitModel;
+import org.opentripplanner.util.TestUtils;
+import org.opentripplanner.util.time.TimeUtils;
+
+public class RaptorPathToItineraryMapperTest {
+
+  private static final int BOARD_COST_SEC = 60;
+  private static final int TRANSFER_COST_SEC = 120;
+  private static final double[] TRANSIT_RELUCTANCE = new double[] { 1.0 };
+  public static final double WAIT_RELUCTANCE = 0.8;
+  private static final int[] STOP_COSTS = { 0, 0, 3_000, 0, 6_000, 0, 0, 0, 0, 0 };
+
+  private static final int ACCESS_START = TimeUtils.time("10:00");
+  private static final int TRANSIT_END = TimeUtils.time("11:00");
+
+  private final TestTransitData data = new TestTransitData();
+
+  public static final CostCalculator<TestTripSchedule> COST_CALCULATOR = new DefaultCostCalculator<>(
+    BOARD_COST_SEC,
+    TRANSFER_COST_SEC,
+    WAIT_RELUCTANCE,
+    TRANSIT_RELUCTANCE,
+    STOP_COSTS
+  );
+
+  @ParameterizedTest
+  @CsvSource({ "0", "3000", "-3000" })
```

Comment:
```suggestion
  @ValueSource({ "0", "3000", "-3000" })
```

I find it strange that it works anyway.

Reply:
I have changed it to use valuesource par your comment - however it has no real effect the test works the same. I dont just a keyvalue pair anymore in this test so the valuesource works better.